### ISSUE TITLE
View and download user summary statistics

### ISSUE THREAD

**Issue by cgiovando** [hotosm/osm-analytics#27]
For any selected AOI and filters set, to have a popup window showing the top 10 (maybe 100?) mappers and their number of edits. Example:

| Username | Buildings | Roads |
|----------|-----------|-------|
| User1    | 1000      | 50    |
| User2    | 500       | 200   |
| ...      | ...       | ...   |

I feel this may be straight forward at feature zoom level, but become a bit more complicated when data is being aggregated at lower zoom levels.

**Comment by tyrasd** [comment]
> straight forward at feature zoom level

Indeed. There's now a modal (opened by clicking on the number of contributors) that gives the list of top x users for the currently selected feature type (e.g. buildings) and the percentage of features last touched by the respective users. However, this data is currently not available in the lower zoom levels, because the uid is not a property that is currently included in the data samples for each grid cell (see https://github.com/hotosm/osm-analytics-cruncher/issues/1).

Another related issue is that even with uid-samples, it turns out to be more tricky than expected to estimate the total number of distinct contributors from a sample. After a quick research, it looks like [Gool-Turing estimation](https://en.wikipedia.org/wiki/Good%E2%80%93Turing_frequency_estimation) could maybe be used for this…

**Comment by iyan31** [comment]
I saw similar functionality on [OSM Reporter](http://osm.kartoza.com/?bbox=20.116653442382812,-34.13113150278439,20.762100219726562,-33.93196649986436&obj=building) built by Kartoza. They also provide the source code for this in [Github](https://github.com/kartoza/osm-reporter) if it may help.
